src/components/Settings/Settings.styles.js

```diff
@@ -0,0 +1,70 @@
+import styled from '@emotion/styled';
+
+import Form from 'components/Form';
+
+export const Screen = styled.div`
+  background-color: ${(props) => props.theme.color.gray200};
+  display: flex;
+  flex-direction: column;
+  min-height: inherit;
+`;
+
+export const Container = styled.div`
+  align-items: center;
+  background-color: ${(props) => props.theme.color.white};
+  border-radius: ${(props) => props.theme.borderRadius.md};
+  display: flex;
+  flex-direction: column;
+  margin: 10px auto;
+  padding: 2rem;
+  width: auto;
+`;
+
+export const SettingsTitle = styled.h2`
+  align-self: center;
+  color: ${(props) => props.theme.color.gray700};
+  font-size: ${(props) => props.theme.fontSize.x3l};
+  font-weight: 300;
+  grid-area: title;
+  margin-bottom: 2.5rem;
+  text-align: center;
+`;
+
+export const SuccessText = styled.p`
+  color: ${(props) => props.theme.color.green700};
+`;
+
+export const FormBody = styled(Form)`
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  margin-top: 1.5rem;
+  width: fit-content;
+`;
+
+export const FormButton = styled(Form.Button)`
+  border-radius: ${(props) => props.theme.borderRadius.full};
+  background-color: ${(props) => props.theme.color.indigo600};
+  color: ${(props) => props.theme.color.white};
+  font-weight: 700;
+  letter-spacing: ${(props) => props.theme.letterSpacing.wide};
+  margin-top: 1rem;
+  margin-bottom: 1rem;
+  padding-right: 1rem;
+  padding-left: 1rem;
+  text-transform: uppercase;
+  width: 16rem;
+
+  &:hover {
+    background-color: ${(props) => props.theme.color.indigo600};
+  }
+
+  &:disabled {
+    background-color: ${(props) => props.theme.color.indigo200};
+    box-shadow: ${(props) => props.theme.boxShadow.none};
+
+    &:hover {
+      background-color: ${(props) => props.theme.color.indigo200};
+    }
+  }
```

Comment:
I just noticed that we were overriding all button styles when we only need to override the width. Check my last PR please y changed some styling tho.

Reply:
i removed the redundant rules, thanks!